### DISCUSSION TITLE
How to use redirectMode?

### DISCUSSION THREAD

**Discussion by FaonDev** [mercadopago/sdk-react#25] [General]
I want to change the default redirect into a Modal. Can i do this?

**Comment by AdaltonLeite** [comment]
Hello @FaonDev,

Absolutely, you can do it! Add **redirectMode** to your initialization settings to ensure proper handling of your redirects. 

Example: 
`<Wallet initialization={{ preferenceId: '<PREFERENCE_ID>', redirectMode: 'modal' }} />`

To obtain more detailed information, please follow the [documentation](https://www.mercadopago.com.ar/developers/en/docs/checkout-bricks/wallet-brick/additional-customization/opening-mode#editor_2) provided. 

I hope that helps, thank you!

**Reply by FaonDev** [reply]
Thanks for answering. But still seeing this error. I'm using version ^0.0.3 (latest) in with TSX.
![image](https://user-images.githubusercontent.com/124811747/230956300-fd715066-3c1a-47c8-bfb5-de45679fcafa.png)
If you can provide more details I would be grateful! 🙂

**Reply by AdaltonLeite** [reply]
@FaonDev, we overlooked adding support in our typescript file. However, we have addressed this issue with the following [PR](https://github.com/mercadopago/sdk-react/pull/26). Once it is released, you can upgrade to version 0.0.6 and this problema will be resolved.

**Reply by FaonDev** [reply]
Perfect. Thanks for answering ❤️! By the way, I managed to solve it just by changing the file extension to JSX. So, a temporary solution.

**Reply by AdaltonLeite** [reply]
@FaonDev, we have just published version **[0.0.6](https://www.npmjs.com/package/@mercadopago/sdk-react/v/0.0.6)**, with it we fixed the bug you reported. I believe that now you won't have this problem anymore. 

I hope that helps, thank you!
